resources/web/wwi/css/toolbar.css

```diff
@@ -113,6 +113,7 @@ webots-view .icon-info        { background-position: -306px -2px }
 webots-view .icon-settings    { background-position: -382px -2px }
 webots-view .icon-fullscreen  { background-position: -420px -2px }
 webots-view .icon-partscreen  { background-position: -458px -2px }
+webots-view .icon-robotWindow { background-position: -654px -2px }
```

Comment:
```suggestion
webots-view .icon-robot-window { background-position: -654px -2px }
```

Reply:
Same reasoning as previous comment, let me know and I can change it.